metrics_sdk/lib/opentelemetry/sdk/metrics/export/console_exporter.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Metrics
+      module Export
+        class ConsoleExporter
+          PREFERRED_TEMPORALITY = 'delta'
+
+          def export(metrics)
+            puts metrics
+          end
+
+          def shutdown
+            SUCCESS
+          end
+
+          def preferred_temporality
```

Comment:
Looks like we also need a way to specify aggregation temporality - i.e. an attribute writer or an initializer argument.

Reply:
Will differ configuration of temporality to a follow up PR.